src/components/Slider/style.module.css

```
@@ -0,0 +1,95 @@
+.root {
+  position: relative;
+  display: flex;
+  width: 100%;
+  gap: 2rem;
+  margin: 2rem 2rem 8rem;
+}
+
+.slider {
+  position: relative;
+  display: flex;
+  align-items: center;
+  user-select: none;
+  touch-action: none;
+  width: 100%;
+  flex-grow: 1;
+}
+
+.thumbWrap {
+  display: flex;
+  align-items: center;
+}
+.track {
+  position: relative;
+  flex-grow: 1;
+  border-radius: 1rem;
+
+  background-color: var(--theme-input-bg-color);
+
+  /* horizontal-specific */
+  height: 0.4rem;
+}
+
+.range {
+  position: absolute;
+  background-color: var(--palette-pink);
+  border-radius: 1rem;
+  height: 100%;
+}
+
+.thumb {
+  --thumb-size: 1.8rem;
```

```suggestion
  --thumb-size: 1.5rem;
```

1.8 is a bit too much IMO